build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,41 @@
+FROM registry.access.redhat.com/ubi8
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Add adoptopenjdk repo
+COPY build/package/yum.repos.d/adoptopenjdk.repo /etc/yum.repos.d/adoptopenjdk.repo
+
+# Install Adoptopenjdk
+RUN yum install -y wget && \
+    INSTALL_PKGS="adoptopenjdk-16-hotspot" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    yum clean all -y && \
+    exactVersion=$(ls -lah /usr/lib/jvm | grep "adoptopenjdk-16-hotspot" | awk '{print $NF}' | head -1) && \
+    alternatives --set java /usr/lib/jvm/${exactVersion}/bin/java && \
+    alternatives --set javac /usr/lib/jvm/${exactVersion}/bin/javac
+
+RUN java -version && javac -version
+
+RUN cd /opt && \
+    wget https://www-eu.apache.org/dist/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.tar.gz && \
```

Comment:
It would be great to move the version into an env variable to the top of the Dockerfile. Also, the documentation does not mention Maven. Do we want to remove Maven or document it? ;)

Reply:
I'm not sure what to do. On one hand is nice to have maven installed in case we need it in the future. On the other hand, the `build-java.sh` does not supports it. It useless to install it if this capability is not supported and documented.

Reply:
Then we remove it. The only advantage to having it in the image is that someone could write a Maven-task using the existing image, as opposed to creating a custom image and task. I think it's better to take it out since you cannot use the task as-is for Maven projects.

Reply:
yes, let us remove it